Client/Frontend/Browser/BrowserLongPressGestureRecognizer.swift

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+import UIKit
+
+enum ElementType {
+    case ImageSrc(NSURL)
+    case hrefLink(NSURL)
+}
+
+protocol BrowserLongPressGestureDelegate: class {
```

Comment:
For consistency with the <code>ReaderMode</code> helper, let's remove the Browser prefix and just call this <code>LongPressGestureDelegate</code>.